docs/using/authentication.rst

```diff
@@ -16,12 +16,14 @@ If you have not already followed these steps, see the :ref:`flocker-node install
 
 .. _generate-api:
 
-Generate an API user certificate
-================================
+Generating an API user certificate
+==================================
 
 The CLI package includes the ``flocker-ca`` program which is used to generate certificate and key files.
 
-You can run ``flocker-ca --help`` for a full list of available commands. For API user certificates, run the ``flocker-ca create-api-certificate`` command from the same directory containing the certificate authority files generated when you first :ref:`installed the cluster <authentication>`.
+.. note:: You can run ``flocker-ca --help`` for a full list of available commands.
+
+For API user certificates, run the ``flocker-ca create-api-certificate`` command from the same directory containing the certificate authority files generated when you first :ref:`installed the cluster <authentication>`.
```

Comment:
"from the same directory containing the certificate authority files" > "from the directory which contains" or something similar.